deebot_client/commands/json/water_info.py

```diff
@@ -48,10 +48,21 @@ class SetWaterInfo(JsonSetCommand):
         {
             "amount": InitParam(WaterAmount),
             "enable": None,  # Remove it as we don't can set it (App includes it)
+            "sweepType": InitParam(SweepType),
         }
     )
 
     def __init__(self, amount: WaterAmount | str) -> None:
         if isinstance(amount, str):
             amount = get_enum(WaterAmount, amount)
         super().__init__({"amount": amount.value})
```

Comment:
```suggestion
```

Please check the CI as it will notify that no two `__init__` functions are allowed